src/utils/httpGet.test.ts

```diff
@@ -0,0 +1,123 @@
+import axios, { AxiosResponse } from 'axios';
+import { httpGet } from './httpGet';
+import { URL_BASE, API_KEY } from '../utils/constants';
+import type { ISectionsResponseResults, ISectionsResponse, ISearchResponseResults, ISearchResponse } from '../types/types';
+
+jest.mock('axios');
+
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+
+describe(`httpGet`, () => {
+    const mockedSectionsResponseResults: ISectionsResponseResults = {
+        id: 'test-section-id',
+        webTitle: 'test-section-webTitle',
+        webUrl: 'test-section-webUrl',
+    };
+
+    const mockedSectionsResponse: ISectionsResponse = {
+        results: [mockedSectionsResponseResults],
+    };
+
+    const mockedAxiosSectionsResponse: AxiosResponse<{ response: ISectionsResponse }> = {
+        data: { response: mockedSectionsResponse },
+        status: 200,
+        statusText: 'OK',
+        headers: {},
+        config: {},
+    };
+
+    const mockedSearchResponseResultsList: ISearchResponseResults[] = Array.from(new Array(21).fill(null).map((_, idx) => (
+        {
+            'id': `test-article-id-${idx}`,
+            type: 'test-article-type',
+            sectionId: 'test-article-sectionId',
+            sectionName: 'test-article-sectionName',
+            webPublicationDate: new Date('2023-01-01').toString(),
+            webTitle: 'test-article-webTitle',
+            webUrl: 'test-article-webUrl',
+        }
+    )));
+
+    const mockedSearchResponse: ISearchResponse = {
+        total: 21,
+        startIndex: 1,
+        pageSize: 10,
+        currentPage: 1,
+        pages: 3,
+        results: mockedSearchResponseResultsList,
+    };
+
+    const mockedAxiosSearchResponse: AxiosResponse<{ response: ISectionsResponse }> = {
+        data: { response: mockedSearchResponse },
+        status: 200,
+        statusText: 'OK',
+        headers: {},
+        config: {},
+    };
+
+    function mockedAxiosGetImplementation(url: string) {
+        return new Promise((resolve, reject) => {
+            if (url === `${URL_BASE}/sections`) {
+                resolve(mockedAxiosSectionsResponse);
+            } else if (url === `${URL_BASE}/search`) {
+                resolve(mockedAxiosSearchResponse);
+            } else {
+                reject(new Error('request error'));
+            }
+        });
+    }
+
+    beforeEach(() => {
+        mockedAxios.get.mockImplementationOnce(mockedAxiosGetImplementation);
+    })
+
+    describe(`call httpGet('sections')`, () => {
+        it(`should resolve with ISectionsResponse type object`, async () => {
```

Comment:
When looking at `httpGet` implementation you would really expect it to work that way or your expect it to work that way based on the docs of The Guardian API? Because when I look at this method I would test two things:
- that it is calling axios's get method with proper params and is returning `response.data.response` from the response it will get from axios's call.